### DISCUSSION TITLE
Hostname/IP does not match question

### DISCUSSION THREAD

**Discussion by oasad92** [iklein99/homebridge-smartthings#230] [Q&A]
Hello, first thank you for the amazing plugin. One day randomly my smartthing devices stopped working and I get this error: 

Error getting devices from Smartthings: Error: Hostname/IP does not match certificate's altnames: Host: api.smartthings.com. is not in the cert's altnames: DNS:mybackoffice.online, DNS:*.mybackoffice.online

Any idea what could have changed or how I can fix this? Thanks in advance!

**Comment by iklein99** [comment]
No idea.  It would seem that there is some wierd stuff with the certificate for api.smartthings.com, but I've never seen this.  Almost seems to me that you are going through some sort of proxy that is interfering with communications to the smartthings endpoint.  Does mybackoffoce.online mean anything to you?

**Comment by oasad92** [comment]
Only thing I can think of is that might be the Samsung SmartThings hub that
I use? But I’ve had that hub for years and have never had this issue until
a week ago so not sure if anything changed recently that might have
affected it.

**Comment by iklein99** [comment]
It isn't the hub.  The plugin is making a call to https://api.smartthings.com, and it is failing because it doesn't like the certificate that is coming back.  It looks to me like something is intercepting the request and returning a bad certificate.  Something odd on your network.
